audit-triplestore-web/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+#
+# The logging properties used
```

Comment:
That is an artifact of the `mvn archtype:generate` command. The audit-triplestore code also uses `slf4j` with a log4j config. Are you suggesting changing both modules?